build.sh

```diff
@@ -21,6 +21,26 @@ if [ -z ${VCPKG_ROOT} ]; then
   exit 1
 fi
 
+# Determine vcpkg os and arch triplet
+OS="unknown"
+ARCH="$(uname -m)"
+
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     OS=linux;;
+    Darwin*)    OS=osx;;
+    *)          echo "Unknown machine ${unameOut}"
+esac
+
+unameOut="$(uname -m)"
+case "${unameOut}" in
+    arm*)       ARCH=arm64;;
```

Comment:
This might be temporary as the triplet on aarch64 might be different. Not entirely sure yet. For now this is ok.

Reply:
:+1: